output/result_.csv

```diff
@@ -1,7 +1 @@
-0,1001 BLATTGRÜN DRAGÉES,0,100,swissmedic,50.0
-1,-LONGORAL,0,100,swissmedic,50.0
-2,-SULFAT,0,100,swissmedic,50.0
-3,-GRANORAL ARÔME MANGUE,0,100,swissmedic,50.0
-4,-GRANORAL ARÔME PÊCHE-ABRICOT,0,100,swissmedic,50.0
-5,-GRANORAL ARÔME CASSIS,0,100,swissmedic,50.0
-6,-ORALEFF,0,100,swissmedic,50.0
+0,No object found,0,,,none,0.0,,0.0,0.0,0.0
```

Comment:
I am glad you removed the outputs! could you add this to the .gitignore so that we do not commit result files in the future to github?

Reply:
done :-)